modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+```
+
+[[mounted-disks]]
+=== Disks are correctly mounted
+
+Ensure that the correct disks are mounted and if multiple devices are used, they are configured as RAID-0. Other RAID configurations can have significantly worse latencies. File system should be of type XFS. If XFS is unavailable, ext4 is an appropriate alternative. 
+
+Example:
+```bash
+grep data_directory /etc/redpanda/redpanda.yaml
+    data_directory: /var/lib/redpanda/data
+df -khT /var/lib/redpanda/data
+```
+
+Output:
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/nvme0n1   xfs   1.8T   14G  1.8T   1% /mnt/vectorized
+```
+
+Output:
+
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/md0       xfs    14T   99G   14T   1% /mnt/vectorized
+```
+
+Output:
+```bash
+mdadm: An option must be given to set the mode before a second device (/dev/md0) is listed
+```
+
+Example:
+```bash
+mdadm --detail /dev/md0
+```
+
+Output:
+```bash
+/dev/md0:
+           Version : 1.2
+     Creation Time : Thu Apr 18 11:03:46 2024
+        Raid Level : raid0
+        Array Size : 14648172544 (13969.59 GiB 14999.73 GB)
+      Raid Devices : 2
+     Total Devices : 2
+       Persistence : Superblock is persistent
+
+       Update Time : Thu Apr 18 11:03:46 2024
+             State : clean
+    Active Devices : 2
+   Working Devices : 2
+    Failed Devices : 0
+     Spare Devices : 0
+
+            Layout : -unknown-
+        Chunk Size : 512K
+
+Consistency Policy : none
+
+              Name : ip-172-31-24-82:0  (local to host ip-172-31-24-82)
+              UUID : e9574118:10d562bf:ed3ca2d9:68ccc3a6
+            Events : 0
+
+    Number   Major   Minor   RaidDevice State
+       0     259        2        0      active sync   /dev/nvme2n1
+       1     259        0        1      active sync   /dev/nvme1n1
+```
+
+[[auth-enable]]
+=== Authentication enabled
+
+Ensure that authentication is set up (or other mitigations are in place). Without SASL authentication enabled, anybody can potentially connect to the Redpanda brokers.
+
+Also ensure that the Admin API is secured, and any users defined in the superusers configuration are appropriately protected with strong credentials.
+
+Example:
+```bash
+rpk cluster config get kafka_enable_authorization
+```
+
+Output:
+```bash
+true
+```
+
+[[tls-enabled]]
+=== TLS Enabled 
+
+Ensure that all public interfaces have TLS enabled.
+
+Example:
+```bash
+grep tls /etc/redpanda/redpanda.yaml
+    kafka_api_tls:
+    admin_api_tls:
+    rpc_server_tls:
+        tls:
+        tls:
+    schema_registry_api_tls:
+```
+
+[[redpanda-tuners]]
+=== Redpanda Tuners
+
+Ensure that tuners have been run on all cluster hosts. Redpanda tuners ensure that the operating system is configured for optimal performance and can have a significant impact on the latency and throughput. In Kubernetes, you may need to run the tuners on the hosts themselves, rather than in containers.
```

Comment:
"...and can have a significant impact on the latency and throughput"
^^ The tuner can have significant impact on latency and throughput...How so?
Shouldn't this minimally link to https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/#tune-the-linux-kernel-for-production or https://docs.redpanda.com/current/reference/rpk/rpk-redpanda/rpk-redpanda-tune/?